mcs/class/corlib/System.Reflection/Module.cs

```diff
@@ -43,97 +43,17 @@ internal enum ResolveTokenError {
 		Other
 	};
 
-	[ComVisible (true)]
-	[ComDefaultInterfaceAttribute (typeof (_Module))]
 	[Serializable]
-	[ClassInterfaceAttribute (ClassInterfaceType.None)]
 	[StructLayout (LayoutKind.Sequential)]
-#if MOBILE
-	public abstract class Module : ISerializable, ICustomAttributeProvider {
-#else
-	public abstract class Module : ISerializable, ICustomAttributeProvider, _Module {
-#endif
-		public static readonly TypeFilter FilterTypeName = new TypeFilter (filter_by_type_name);
-		public static readonly TypeFilter FilterTypeNameIgnoreCase = new TypeFilter (filter_by_type_name_ignore_case);
-	
-		const BindingFlags defaultBindingFlags = 
-			BindingFlags.Public | BindingFlags.Static | BindingFlags.Instance;
-
-		protected
-		Module () {
-		}
-
-		public ModuleHandle ModuleHandle {
-			get {
-				return new ModuleHandle (GetImpl ());
-			}
-		}
-
+	public abstract partial class Module : ISerializable, ICustomAttributeProvider {
```

Comment:
Could be just simple `partial class Module` there rest is copied from corefx part